.github/workflows/expo-eas-build.yml

```diff
@@ -28,7 +28,8 @@ jobs:
       - uses: expo/expo-github-action@v7
         with:
           expo-cache: true
-          expo-version: 5.x
+          expo-version: latest
+          eas-version: latest
           token: ${{ secrets.expo-token }}
       - run: yarn
-      - run: npx eas build --platform ${{ inputs.platform }} --non-interactive --profile=${{ inputs.profile }} --no-wait ${{ inputs.submit && '--auto-submit' ||  ''}}
+      - run: eas build --platform ${{ inputs.platform }} --non-interactive --profile=${{ inputs.profile }} ${{ inputs.submit && '--auto-submit' ||  ''}}
```

Comment:
do we still need the extra input(s) such as `submit`?

Reply:
they are not used now. I left them just in case we want to change the flow and use them in a different way.